base/Dockerfile

```
@@ -3,10 +3,10 @@ FROM public.ecr.aws/spacelift/runner-terraform
 USER root
 WORKDIR /home/spacelift
 
-RUN apk -U upgrade && apk add --no-cache \
-    ansible \
-    py3-pip
+# Pinning to 7 for now, will introduce more tags a bit later
 
-RUN pip install ansible-runner
+RUN apk -U upgrade && apk add --no-cache gcc py3-pip python3-dev musl-dev libffi-dev && \
+  pip install ansible==7.* ansible-runner==2.* --break-system-packages && \
```

```suggestion
  pip install ansible==7.* ansible-runner==2.* && \
```

This works for me locally

---

let me see

---

Nevermind, my runner-terraform image was not up to date 🤦🏻